src/components/workflowEditor/types.ts

```diff
@@ -62,6 +62,13 @@ export interface NoGitOpsConfiguredWarningType {
     closePopup: (isContinueWithHelm: boolean) => void
 }
 
+interface MatchType {
```

Comment:
no need of custom match type use `RouteComponentProps` to include types for all router props